Cargo.toml

```diff
@@ -28,8 +28,8 @@ zeroize = { version = "1.1.0", features = ["zeroize_derive"] }
 env_logger = "0.7.1"
 
 [build-dependencies]
-bindgen = "0.54.0"
-pkg-config = "0.3.17"
+bindgen = "0.55.1"
```

Comment:
`bindgen` can not be upgraded to `0.55.1` because it pulls in `clang-sys` version `1.0.0` which itself pulls in `libloading` version `0.6.3` which uses the `non_exhaustive` attribute which is not supported by `rustc` version `1.38.0` that we test on CI.
If we decide to increase the version of `bindgen` it means that our minimum supported Rust compiler would be [`1.40.0`](https://blog.rust-lang.org/2019/12/19/Rust-1.40.0.html#non_exhaustive-structs-enums-and-variants)